go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	cosmossdk.io/math v1.3.0
 	filippo.io/edwards25519 v1.1.0
 	github.com/cometbft/cometbft v0.37.5
-	github.com/cometbft/cometbft-db v0.8.0
+	github.com/cometbft/cometbft-db v0.9.1
```

Comment:
Is there any specific action required for the underlying bump here?

I'd assume upgrading to the next major version might require software changes.

If not, that's fine.

Reply:
fee-abs and feemarket are using this version may it is getting overridden.